v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/KafkaReadOptions.java

```diff
@@ -49,23 +49,49 @@ public interface KafkaReadOptions extends PipelineOptions {
 
   void setKafkaReadTopics(String inputTopics);
 
-  @TemplateParameter.Enum(
+  @TemplateParameter.Boolean(
       order = 3,
       groupName = "Source",
+      name = "enableCommitOffSets",
+      optional = true,
+      description = "Enable committing offsets to Kafka",
```

Comment:
I suggest renaming this to "Commit Offsets to Kafka" (keep labels short and capitalized).

Reply:
Done